package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "5.6.0",
+  "version": "5.7.0",
```

Comment:
This is technically a breaking change but since this is an unreleased feature, we can get away with a minor here I'm p sure cc @chris-stytch